src/main/java/edu/kis/powp/jobs2d/command/DriverCommand.java

```diff
@@ -2,10 +2,12 @@
 
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+import java.io.Serializable;
+
 /**
  * DriverCommand interface.
  */
-public interface DriverCommand {
+public interface DriverCommand extends Cloneable, Serializable {
```

Comment:
Please, write in comment what is Serializable interface for (in this task)

Reply:
I have used this in 30 line in  CommandFeature.deepCopyDriverCommand

> return SerializationUtils.clone(driverCommandToCopy);

I had to implement Serializable to use SerializationUtils because clone in this library as parametr has <T object>
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SerializationUtils.html